Makefile

```diff
@@ -46,12 +46,30 @@ install:
 	# see https://github.com/systemd/systemd/blob/v247/src/shared/clock-util.c#L145
 	touch $(DESTDIR)/usr/lib/clock-epoch
 
+	# generate the changelog, for this we need the previous core snap
+	# to be installed, this should be handled in snapcraft.yaml
+	if [ -e "/snap/$(SNAP_NAME)/current/usr/share/snappy/dpkg.yaml" ]; then \
```

Comment:
It could be good to show a warning to the terminal in case this is not the case

Reply:
nice idea, added